cves/CVE-2017-7659.yml

```diff
@@ -220,4 +232,9 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
```

Comment:
Unable to leave a comment directly on any of the `lessons` fields directly, so I'll just place it here.

Perhaps `defense in depth` applies? Could probably argue that if we're using a separate, complex module for performing some action, maybe we need another layer of validation around it to ensure that it doesn't crash/return garbage data?

Reply:
I think that's a good point, I was on the fence about arguing for defense in depth for this specific CVE but im glad someone else thinks so too.